proposals/Template-Instantiation.md

```diff
@@ -446,12 +446,9 @@ The  `createInstance(optional any state)` method on `HTMLTemplateElement`, when
 
 > Note: We run the concepts to _apply attribute template part list_ and _apply node template part list_ immediately to strip away the mustache syntax in the original template as well as whitespaces before & after it to keep the initial template state consistent with the one after running these concepts in a template process callback. Actual implementations can run these algorithm as it clones the tree, and avoid unnecessary churn of text nodes and strings as an optimization.
 
-When there is exactly one `{{~}}` inside a template, we keep
```

Comment:
Maybe this incomplete sentence should be completed instead of removed

Reply:
Removing it is fine for now.